frontend/app/tests/unit/specs/composables/user/account.spec.ts

```diff
@@ -82,7 +82,8 @@ describe('composables::user/account', () => {
       });
 
       await createNewAccount({
-        credentials: { username: 'test', password: '1234' }
+        credentials: { username: 'test', password: '1234' },
+        initialSettings: { submitUsageAnalytics: true }
```

Comment:
hey @Jaspreet-singh-1032 do you think it would be possible to change the default here to false and verify that false is properly passed through? since that was the bug?